2-mandatory/3-stocks.js

```diff
@@ -48,7 +64,13 @@ function getAveragePrices(closingPricesForAllStocks) {
     The price change value should be rounded to 2 decimal places, and should be a number (not a string)
 */
```

Comment:
Nice!
One question: Will this code - `closingPricesEachcompany[4] - closingPricesEachcompany[0]` - still work if we have more than 5 prices in each array? Or less than 5 prices?
Can you think of how to make sure this works for any numbers of prices?